src/Application/Grouper/MethodGrouper.php

```diff
@@ -51,7 +51,7 @@ public function groupMethods(array $methods): array
 
                 if (isset($methods[$setMethod])) {
                     $type = 'get_set';
-                    $name = 'testSetGet' . \substr($method, 3);
+                    $name = 'test' . \substr($method, 3);
```

Comment:
this is currently testing `SetGet` thats why it is named this way. It should be as explicit as possible as there could be other methods for that attribute.